### ISSUE TITLE
DVD Menu not clickable

### ISSUE THREAD

**Issue by toutestneutral** [jaruba/wcjs-player#35]
I try to read a DVD via wcjs-player but when it starts I can't hover or click on the menu elements

Is it possible to activate this functionality? if not, is there a way to call the corresponding method from vlclib?

Thanks!

**Comment by RSATom** [comment]
Unfortunately there are no way to pass mouse/keyboard events to libvlc with our current approach of using libvlc (we use vmem plugin). I think the only way solve this problem is write custom libvlc plugin. And It's pretty complex task.

**Comment by toutestneutral** [comment]
Can't we patch the plugin using this method? https://www.videolan.org/developers/vlc/doc/doxygen/html/group__libvlc__video.html#gad53def725dd5a50a3823b68ed31bca1a

**Comment by RSATom** [comment]
No, it's enable/disable mouse input only if render window created by libvlc itself. In our case there are no any window created on libvlc side.

**Comment by RSATom** [comment]
I've started work on my own vmem plugin to solve many issues with libvlc's implementation. Maybe some time late this feature will be available too. But can't promise when.
